src/current/v24.1/migrate-from-oracle.md

```diff
@@ -325,11 +325,11 @@ The last phase of the migration process is to change the [transactional behavior
+Both Oracle and CockroachDB support [multi-statement transactions]({% link {{ page.version.version }}/transactions.md %}), which are atomic and guarantee ACID semantics. However, CockroachDB operates under [`SERIALIZABLE`]({% link {{ page.version.version }}/demo-serializable.md %}) isolation by default, while Oracle defaults to [`READ COMMITTED`]({% link {{ page.version.version }}/read-committed.md %}), which can create both [non-repeatable reads and phantom reads]({% link {{ page.version.version }}/read-committed.md %}#non-repeatable-reads-and-phantom-reads) when a transaction reads data twice. It is typical that Oracle developers will use `SELECT FOR UPDATE` to work around `READ COMMITTED` concurrency anomalies. Both the [`READ COMMITTED`]({% link {{ page.version.version }}/read-committed.md %}) isolation level and the [`SELECT FOR UPDATE`]({% link {{ page.version.version }}/select-for-update.md %}) statement are supported in CockroachDB.
 
 Regarding locks, Cockroach utilizes a [lightweight latch]({% link {{ page.version.version }}/architecture/transaction-layer.md %}#latch-manager) to serialize access to common keys across concurrent transactions. Oracle and CockroachDB transaction control flows only have a few minor differences; for more details, refer to [Transactions - SQL statements]({% link {{ page.version.version }}/transactions.md %}#sql-statements).
 
-As CockroachDB does not allow serializable anomalies, [transactions]({% link {{ page.version.version }}/begin-transaction.md %}) may experience deadlocks or [read/write contention]({% link {{ page.version.version }}/performance-best-practices-overview.md %}#transaction-contention). This is expected during concurrency on the same keys. These can be addressed with either [automatic retries]({% link {{ page.version.version }}/transactions.md %}#automatic-retries) or [client-side transaction retry handling]({% link {{ page.version.version }}/transaction-retry-error-reference.md %}#client-side-retry-handling).
+As CockroachDB does not allow serializable anomalies under `SERIALIZABLE` isolation, [transactions]({% link {{ page.version.version }}/begin-transaction.md %}) may experience deadlocks or [read/write contention]({% link {{ page.version.version }}/performance-best-practices-overview.md %}#transaction-contention). This is expected during concurrency on the same keys. These can be addressed with either [automatic retries]({% link {{ page.version.version }}/transactions.md %}#automatic-retries) or [client-side transaction retry handling]({% link {{ page.version.version }}/transaction-retry-error-reference.md %}#client-side-retry-handling).
```

Comment:
```suggestion
Because CockroachDB does not allow serializable anomalies under `SERIALIZABLE` isolation, [transactions]({% link {{ page.version.version }}/begin-transaction.md %}) may experience deadlocks or [read/write contention]({% link {{ page.version.version }}/performance-best-practices-overview.md %}#transaction-contention). This is expected during concurrency on the same keys. These can be addressed with either [automatic retries]({% link {{ page.version.version }}/transactions.md %}#automatic-retries) or [client-side transaction retry handling]({% link {{ page.version.version }}/transaction-retry-error-reference.md %}#client-side-retry-handling).
```
"during concurrency" seems a bit strange.